index.html

```diff
@@ -0,0 +1,740 @@
+                <div class="chip">
+                    <p>All</p>
+                </div>
+                <div class="chip">
+                    <p>Microverse</p>
+                </div>
+                <div class="chip">
+                    <p>Programing Languages</p>
+                </div>
+                <div class="chip">
+                    <p>Coding</p>
+                </div>
+                <div class="chip">
+                    <p>Web Design</p>
+                </div>
+                <div class="chip">
+                    <p>Css</p>
+                </div>
+                <div class="chip">
+                    <p>Software Development</p>
+                </div>
+                <div class="chip">
+                    <p>Coding School</p>
+                </div>
+                <div class="chip">
+                    <p>Tech</p>
+                </div>
+                <div class="chip">
+                    <p>UI/UX</p>
+                </div>
+                <div class="chip">
+                    <p>Web Design</p>
+                </div>
+                <div class="chip">
+                    <p>Microverse Online</p>
+                </div>
+
+            </div>
+            <div class="video-container">
+                <div class="video-content-cover">
+                    <div class="video-content">
+                        <a href="#" class="video-box">
+                           <iframe width="360" height="145" src="https://www.youtube.com/embed/kUMe1FH4CHE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                        </a>
+                        <div class="video-details">
+                            <div class="channel-logo">
+                                <img src="images/Channel logo.jpeg" alt="">
+                            </div>
+                            <div class="detail">
+                                <h3 class="title">
+                                    10 Ways to get into trouble tutorial
+                                </h3>
+                                <div class="channel-name">
+                                    Grand Rookies
+                                </div>
+                                <div class="views-upload">
+                                    <div class="views">8M Views</div>
+                                    <div class="upload">2 weeks ago</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="hidden-content">
+                            <div class="btn">
+                                <i class="ri-time-line"></i>
+                                Watch Later
+                            </div>
+                            <div class="btn">
+                                <i class="ri-play-list-2-line"></i>
```

Comment:
I suggest the Use section tag in this case over divs.

Reply:
- [ ] Kindly, when you give a code review, be polite and explain why you ask to change to the use of semantic tags. It would be better to say, '' I think it would be better to use a semantic tags `section`  instead of `div` to make our code clear and easy to maintain, our page accessible, and improve SEO positioning. For further detail check[ this link](https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0)" :champagne: